mgmt/rest/v1/rbody/body.go

```diff
@@ -32,6 +32,8 @@ import (
 	"github.com/urfave/negroni"
 )
 
+const deprecationInfo = "The client API V1 has been depricated. Find more information here: https://github.com/intelsdi-x/snap/issues/1637"
```

Comment:
To be more useful to the user the message should something like this: "The client API v1 has been deprecated, and will be removed <provide date or release here>. Please switch to API v2."

Is API v2 ready to be exposed to end users?